src/pages/index.js

```diff
@@ -135,10 +139,137 @@ class Home extends Component {
-            dangerouslySetInnerHTML={{__html: data.markdownRemark.html}}
-          />
+          <div css={[sharedStyles.markdown, markdownStyles]}>
+            <section className="home-section" css={{
+              [media.lessThan('medium')]: {
+                marginTop: 0,
+                marginBottom: 0,
+                overflowX: 'auto',
+                paddingTop: 30,
+                WebkitOverflowScrolling: 'touch',
+                position: 'relative',
+                maskImage:
+                  'linear-gradient(to right, transparent, white 10px, white 90%, transparent)',
+              },
+            }}>
+              <div css={{
+                display: 'flex',
+                flexDirection: 'row',
+
+                [media.lessThan('medium')]: {
+                  display: 'block',
+                  whiteSpace: 'nowrap',
+                },
+              }}>
+                {marketingColumns.map((column, index) =>
+                  <div key={index} css={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    flex: '0 1 33%',
+                    marginLeft: 40,
+
+                    '&:first-of-type': {
+                      marginLeft: 0,
+
+                      [media.lessThan('medium')]: {
+                        marginLeft: 10,
+                      },
+                    },
+
+                    [media.lessThan('medium')]: {
+                      display: 'inline-block',
+                      verticalAlign: 'top',
+                      marginLeft: 0,
+                      whiteSpace: 'normal',
+                      width: '75%',
+                      marginRight: 20,
+                      paddingBottom: 40,
+
+                      '&:first-of-type': {
+                        marginTop: 0,
+                      },
+                    },
+
+                    '& p': {
+                      lineHeight: 1.7,
+                    },
+                  }}>
+                    <h3 css={{
+                      '&&': { // Make specificity higher than the site-wide h3 styles.
+                        color: colors.subtle,
+                        marginBottom: 20,
+                        paddingTop: 0,
+                        fontWeight: 300,
+                        fontSize: 20,
+
+                        [media.greaterThan('xlarge')]: {
+                          fontSize: 24,
+                          fontWeight: 200,
+                        },
+                      },
+                    }}>{column.title}</h3>
+                    <div dangerouslySetInnerHTML={{__html: column.content}} />
+                  </div>
+                )}
+              </div>
+            </section>
+            <hr css={{
+              height: 1,
+              marginBottom: -1,
+              border: 'none',
+              borderBottom: `1 solid ${colors.divider}`,
+            }} />
+            <section className="home-section">
```

Comment:
Will refactor the examples section in a later PR. Please bear with the hardcoding of class names for now.